src/modules/clusters/cluster-list/cluster-list.html

```diff
@@ -49,28 +49,26 @@ <h1>No Clusters Detected</h1>
-                    
                     <div class="dropdown pull-right dropdown-kebab-pf " ng-show="clusterCntrl.showKebabMenu(cluster)">
-                        <button class="btn btn-link dropdown-toggle"  ng-class="{'disable-pointer': cluster.currentStatus === 'in_progress'}" type="button" id="dropdownKebabRight{{$index}}" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+                        <button class="btn btn-link dropdown-toggle" ng-class="{'disable-pointer': cluster.currentStatus === 'in_progress'}" type="button" id="dropdownKebabRight{{$index}}" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
                             <span class="fa fa-ellipsis-v"></span>
                         </button>
                         <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownKebabRight{{$index}}">
-                            <li ng-show = "cluster.managed === 'Yes'">
-                                <a ng-class="{'disable-pointer': $root.userRole === 'limited' || clusterCntrl.disProfilingBtnClicked}" ng-click="clusterCntrl.doProfilingAction($event, cluster, 'Disable', cluster.clusterId)" uib-tooltip="Disable Profiling" ng-show="cluster.isProfilingEnabled === 'Enabled' || cluster.isProfilingEnabled === 'Mixed'">Disable Profiling
+                            <li ng-show="cluster.managed === 'Yes'">
+                                <a ng-class="{'disable-pointer': $root.userRole === 'limited' || cluster.disableAction || (cluster.jobType === 'EnableDisableVolumeProfiling' && cluster.currentStatus === 'in_progress')}" ng-click="clusterCntrl.doProfilingAction($event, cluster, 'Disable', cluster.clusterId)" uib-tooltip="Disable Profiling" ng-show="cluster.isProfilingEnabled === 'Enabled' || cluster.isProfilingEnabled === 'Mixed'">Disable Profiling
                                     <i class="glyphicon glyphicon-refresh glyphicon-refresh-animate" data-ng-class="{'hidden': clusterCntrl.disProfilingBtnClicked !== true }"></i>
                                 </a>
-                                <a ng-class="{'disable-pointer': $root.userRole === 'limited' || clusterCntrl.enProfilingBtnClicked}" ng-click="clusterCntrl.doProfilingAction($event, cluster, 'Enable', cluster.clusterId)" uib-tooltip="Enable Profiling" ng-show="cluster.isProfilingEnabled === 'Disabled' || cluster.isProfilingEnabled === 'Mixed'">Enable Profiling
+                                <a ng-class="{'disable-pointer': $root.userRole === 'limited' || cluster.disableAction || (cluster.jobType === 'EnableDisableVolumeProfiling' && cluster.currentStatus === 'in_progress')}" ng-click="clusterCntrl.doProfilingAction($event, cluster, 'Enable', cluster.clusterId)" uib-tooltip="Enable Profiling" ng-show="cluster.isProfilingEnabled === 'Disabled' || cluster.isProfilingEnabled === 'Mixed'">Enable Profiling
```

Comment:
can you call a function, it is difficult for someone to understand

$root.userRole === 'limited' || cluster.disableAction || (cluster.jobType === 'EnableDisableVolumeProfiling' && cluster.currentStatus === 'in_progress')

Reply:
@cloudbehl Calling a function was not doing the 2 way binding thing. Hence, need to do this way.